pkg/registry/apis/secret/register.go

```diff
@@ -64,8 +68,12 @@ func NewSecretAPIBuilder(
 	accessClient claims.AccessClient,
 	decryptersAllowList map[string]struct{},
 	isDevMode bool, // REMOVE ME
-) *SecretAPIBuilder {
+) (*SecretAPIBuilder, error) {
 	database := database.New(db)
+	keepers, err := keeperService.GetKeepers()
+	if err != nil {
```

Comment:
Should we panic here if we cant get the keepers? (using the assert pkg)

Reply:
Should we just ignore it for now since it'll be changed based on your PRs?